iambic/config/wizard.py

```diff
@@ -193,6 +197,22 @@ def set_required_text_value(human_readable_name: str, default_val: str = None):
             print("Please enter a valid response.")
 
 
+def set_required_secure_text_value(
+    human_readable_name: str,
+    default_val: Optional[SecretStr] = None,
+):
+    while True:
+        if response := questionary.text(
+            human_readable_name,
+            default=SecretStr(
```

Comment:
Do you happen to know whether SecretStr supports ""?

Reply:
mmm the issue is that `"" `does not support `get_secret_value`

Reply:
perhaps `default_val or ""` is a better approach